### DISCUSSION TITLE
How to be a lecturer

### DISCUSSION THREAD

**Discussion by giscus** [krehwell/P-Blog#1] [General]
# How to be a lecturer

How to be a lecturer? very easy, just take a phd and never make anything on something then act like you are a pro in the field in front of student

http://localhost:3000/blog/be-a-lecturer-in-an-easy-way?giscus=f87cdfd06fff5b63d96781bd6a%2By1hUajXThPA7oHgJoXkPDWJDoXawzz9gnPkbrY6RNm4CvPvE3sYEEH5jpWn8trmBriXV1QcV5bLGhyD8CRoLi9gOJV2hvMyzzzEAK84TSCImsl3Mzvl%2BH9iY%3D

**Comment by Johanshah** [comment]
save this shit later for the sake of future generation. I can't agree more👍🏻
